Client/JsonLDClient.php

```diff
@@ -36,10 +37,10 @@ class JsonLDClient
      * @param CacheInterface|null $cache
      */
     public function __construct(
-        ClientInterface $client,
+        ClientInterface     $client,
         SerializerInterface $serializer,
-        MappingCollection $mappings,
-        ?CacheInterface $cache
+        MappingCollection   $mappings,
+        ?CacheInterface     $cache
```

Comment:
this is actually against the style guide, can you revert please?

Reply:
Ok, done